maxitg/WSS19
# Initial essay draft

## Changes

* Adds an initial version of the computational essay.
* The notebook file is linted with [SaveReadableNotebook](https://resources.wolframcloud.com/FunctionRepository/resources/SaveReadableNotebook) resource function.

## Tests and commands

* Build (evaluate) the essay as outlined in readme.
* Check if all input cells can be evaluated successfully.
